dsc/workflows/base/simple_csv.py

```diff
@@ -95,7 +95,12 @@ def item_metadata_iter(
             for _, row in metadata_df.iterrows():
                 yield row.to_dict()
 
-    def prepare_batch(self) -> tuple[list, ...]:
+    def prepare_batch(
```

Comment:
I think a docstring would be helpful for `SimpleCSV.prepare_batch()`, kind of explaining what it does and doesn't do, and how it thinks about synced vs non-synced batches.

I think this docstring provides an opportunity to explain why `synced` is not used in the method.

Reply:
Also agree!